images/chaos-daemon/Dockerfile

```diff
@@ -1,36 +1,21 @@
-FROM debian:buster-slim
+FROM alpine/curl:3.14 as binaryimage
 
 ARG HTTPS_PROXY
 ARG HTTP_PROXY
 
 ENV http_proxy $HTTP_PROXY
 ENV https_proxy $HTTPS_PROXY
 
-RUN mkdir -p /usr/share/man/man1 /usr/share/man/man2
-
-RUN apt-get update && \
-    apt-get install -y tzdata iptables ebtables net-tools ipset stress-ng iproute2 fuse util-linux procps curl openjdk-11-jre && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
-RUN update-alternatives --set ebtables /usr/sbin/ebtables-legacy
-
-ENV RUST_BACKTRACE 1
 ARG TARGET_PLATFORM=amd64
 
-# install byteman and set env
-RUN curl -L https://mirrors.chaos-mesh.org/byteman-chaos-mesh-download-v4.0.20-0.12.tar.gz -o /usr/local/byteman.tar.gz
-RUN tar xvf /usr/local/byteman.tar.gz -C /usr/local
-RUN mv /usr/local/byteman-chaos-mesh-download-v4.0.20-0.12 /usr/local/byteman
-RUN rm /usr/local/byteman.tar.gz
-
-ENV BYTEMAN_HOME /usr/local/byteman
-ENV PATH $PATH:/usr/local/byteman/bin
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
-ENV PATH $PATH:/usr/local/bin
+RUN mkdir -p /tmp/bin && \
```

Comment:
`RUN mkdir -p /tmp/byteman`? This directory will be used at L14, otherwise, L14 will return an 'Cannot open: No such file or directory` error.

Reply:
@johnhowlett PTAL